packages/modules/imodel-browser/src/containers/iModelThumbnail/useIModelThumbnail.ts

```diff
@@ -63,6 +64,7 @@ export const useIModelThumbnail = (
         .catch((e) => {
           if (e.name === "AbortError") {
             // Aborting because unmounting is not an error, swallow.
+            setThumbnail(defaultIModelThumbnail);
```

Comment:
When aborting, we shouldnt cause a rerender here. From the mentioned issue, it seems like there should be a 401 error or something like that ? (Which shouldnt even get in that branch, so it's a bit weird)

I'm not sure how to test it either, can you reach out directly and I can set up an iModel that will fail like your's do  in one of my projects maybe ?

Reply:
@raplemie 
The API call for thumbnail throws error 404 as : -

![image](https://github.com/iTwin/admin-components-react/assets/97873094/b457b928-ee21-463a-bc5c-63a8444119dc)

This happens when there is no permission to user.
In our consumer project we have such role as Validate.

And then we can reproduce this.

I think we will connect that would be great.